mpc_follower_wrapper/launch/mpc_follower_wrapper.launch

```diff
@@ -8,19 +8,22 @@
         <remap from="steer_lim_deg" to="/vehicle_steer_lim_deg"/>
         <remap from="steering_gear_ratio" to="/vehicle_steering_gear_ratio"/>
 
-        <include file="$(find mpc_follower)/launch/mpc_follower.launch">
-                <arg name="qp_solver_type" value="unconstraint_fast"/>
-                <arg name="mpc_prediction_horizon" value="70.0"/>
-                <arg name="mpc_prediction_sampling_time" value="0.1"/>
-                <arg name="mpc_weight_lat_error" value="0.1"/>
-                <arg name="mpc_weight_terminal_lat_error" value="0.15"/>
-                <arg name="mpc_weight_terminal_heading_error" value="0.03"/>
-                <arg name="mpc_weight_heading_error" value="0.01"/>
-                <arg name="mpc_weight_heading_error_squared_vel_coeff" value="0.1"/>
-        </include>
+        <node pkg="mpc_follower" type="mpc_waypoints_converter" name="mpc_waypoints_converter" output="screen">
+			<remap from="/base_waypoints" to="base_waypoints"/>
+			<remap from="/closest_waypoint" to="closest_waypoint"/>
+			<remap from="/final_waypoints" to="carma_final_waypoints"/>
+		</node>
+
+		<node pkg="mpc_follower" type="mpc_follower" name="mpc_follower" output="screen">
+			<remap from="/ctrl_raw" to="ctrl_raw"/>
+			<remap from="/twist_raw" to="twist_raw"/>
+            <rosparam command="load" file="$(arg vehicle_calibration_dir)/mpc_follower/calibration.yaml"/>
```

Comment:
You need to add this vehicle_calibration_dir argument as a arg tag at the top of this launch file.